content/en/docs/generating-objects/fixture-monkey.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "FixtureMonkey"
+images: []
+menu:
+docs:
+parent: "generating-objects"
+identifier: "fixturemonkey"
+weight: 10
+---
+
+To generate test fixtures, the first step is to create a `FixtureMonkey` instance, which facilitates the creation of test fixtures.
+
+You can use the `create()` method to generate a `FixtureMonkey` instance with default options.
+
+{{< tabpane persist=false >}}
+{{< tab header="Java" lang="java">}}
+
+FixtureMonkey fixtureMonkey = FixtureMonkey.create();
+
+{{< /tab >}}
+{{< tab header="Kotlin" lang="kotlin">}}
+
+val fixtureMonkey = FixtureMonkey.create()
```

Comment:
How about providing guide using `KotlinPlugin()` as it's a required in the Kotlin env